client/extensions/woocommerce/state/ui/payments/methods/README.md

```diff
@@ -0,0 +1,58 @@
+Payment Methods
+================
+
+This module is used to manage payment method related UI.
+
+## Actions
+
+### `openPaymentMethodForEdit( siteId: number, id: string )`
```

Comment:
It's not really JSDoc, just indicating what the parameters for this function are. You don't need to copy in parameter or return descriptions, or do anything more complicated than a short description (1 sentence is fine). There isn't _technically_ a format for this, more than "follow calypso", so feel free to leave out the parameter types in your docs if you want. To me, the important thing is that there's an easy way to see the structure and exported functions in one place.